redis-cluster-activesupport.gemspec

```diff
@@ -19,8 +19,8 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "redis-activesupport"
-  spec.add_development_dependency "bundler", "~> 1.15"
+  spec.add_dependency "activesupport", "~> 6.0"
```

Comment:
What's the thinking with locking this to 6 vs >= 5.2?

Reply:
My thinking is that we won't have a chance to really vet this in Rails 7 yet, but will in Rails 6, so I wanted to be conservative in what's permitted.  The old `redis-activesupport` gem was supported up to 5.2, so 6.0 seemed like a reasonable floor to start from.